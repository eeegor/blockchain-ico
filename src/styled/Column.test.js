import React from 'react';
import renderer from 'react-test-renderer';
import 'jest-styled-components';
import { Column } from '.';

describe('Column', () => {
	it('renders column wrapper', () => {
		const tree = renderer.create(<Column>Info</Column>).toJSON();
		expect(tree).toMatchSnapshot();
	});

	it('renders column wrapper with max width', () => {
		const tree = renderer
			.create(<Column maxWidth="100">Info</Column>)
			.toJSON();
		expect(tree).toMatchSnapshot();
		expect(tree).toHaveStyleRule('max-width', '100px !important');
	});
});
