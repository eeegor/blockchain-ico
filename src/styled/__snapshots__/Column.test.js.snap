// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Column renders column wrapper 1`] = `
<div
  className=""
>
  Info
</div>
`;

exports[`Column renders column wrapper with max width 1`] = `
.c0 {
  max-width: 100px !important;
}

<div
  className="c0"
>
  Info
</div>
`;
